migrations/default/1_create_tables/up.sql

```diff
@@ -0,0 +1,157 @@
+drop schema if exists db cascade;
+create schema db;
+
+-------------------------------------------
+-------- lookup tables --------------------
+-------------------------------------------
+create table db.road_types (
+    id integer primary key,
+    description text not null,
+    owner text
+);
+
+create table db.unit_types (
+    id integer primary key,
+    description text not null,
+    owner text not null
+);
+
+insert into db.road_types (id, description, owner) values
+(1, 'alley', 'cris'),
+(2, 'collector', 'cris'),
+(3, 'arterial', 'cris'),
+(4, 'highway', 'cris'),
+(5, 'other', 'cris'),
+(90000, 'urban trail', 'vz');
+
+-- the check constraints prevent us from creating cris-owned values
+-- in our protected vz namespace. we will add additional constraints
+-- to the _cris tables to prevent cris from using our vz lookup values
+alter table db.road_types add constraint road_type_owner_check
+check ((id < 90000 and owner = 'cris') or (id >= 90000 and owner = 'vz'));
+
+insert into db.unit_types (id, description, owner) values
+(1, 'vehicle', 'cris'),
+(2, 'pedestrian', 'cris'),
+(3, 'motorcycle', 'cris'),
+(4, 'truck', 'cris'),
+(5, 'bicycle', 'cris'),
+(6, 'other', 'cris'),
+(90000, 'e-scooter', 'vz');
+
+alter table db.unit_types add constraint unit_type_owner_check
+check ((id < 90000 and owner = 'cris') or (id >= 90000 and owner = 'vz'));
+
+-------------------------------------------
+-------- Crash tables ---------------------
+-------------------------------------------
+create table db.crashes_cris (
+    crash_id integer primary key,
+    road_type_id integer references db.road_types (id)
+);
+
+-- this prevents cris from using a value that falls within
+-- our custom lookup value namespace
+alter table db.crashes_cris add constraint cris_road_type_chk
+check (road_type_id < 90000);
+
+create table db.crashes_vz (
+    crash_id integer primary key not null references db.crashes_cris (
+        crash_id
+    ) on delete cascade on update restrict,
+    road_type_id integer references db.road_types (id)
+);
+
+create table db.crashes (
+    crash_id integer unique not null references db.crashes_cris (
+        crash_id
+    ) on delete cascade on update restrict,
+    road_type_id integer references db.road_types (id)
+);
+
+
+-------------------------------------------
+-------- Unit tables ----------------------
+-------------------------------------------
+create table db.units_cris (
+    id serial primary key,
```

Comment:
although i think `id` is the most conventional name for this column, i wonder if we might want to use `unit_id` and `person_id` as the PK column names for these tables, just to be consistent with `crash_id`. 

i think i prefer `id`, because these ID columns are custom to our db and that name makes it more clear?

Reply:
just to clarify, `unit_id` does not exist in the CRIS data.

@frankhereford i'm not following if my use of `id` columns matches your description or if you're suggesting a change.